pkg/genevalogging/genevalogging.go

```diff
@@ -110,6 +110,27 @@ const (
 	DB /var/lib/fluent/audit
 	Parser audit
 
+[FILTER]
+    Name nest
+    Match *
+    Operation lift
+    Nested_under user
+    Add_prefix user_
+
+[FILTER]
+    Name nest
+    Match *
+    Operation lift
+    Nested_under responseStatus
+    Add_prefix response_
```

Comment:
let's always match Nested_under and Add_prefix - so in this case `Add_prefix responseStatus_`

Reply:
done